server/app.ts

```diff
@@ -16,7 +16,7 @@ import setUpValidationMiddleware from './middleware/setUpValidationMiddleware'
 import indexRoutes from './routes'
 import productRoutes from './routes/products'
 import componentRoutes from './routes/components'
-import componentEnvRoutes from './routes/componentsEnvDetails'
+import componentEnvironmentRoutes from './routes/componentsEnvDetails'
```

Comment:
This was for everything where you're using `componentsEnv`, including the filenames etc. Just makes it clear.

Reply:
Updated all file names and references to componentsEnvironment